pkg/87d07153-809f-4056-9d1c-5fbbc1dc8a95/package.lp

```diff
@@ -0,0 +1,271 @@
+(librepcb_package 87d07153-809f-4056-9d1c-5fbbc1dc8a95
+ (name "DFN50P200X300X90-8T175X145")
+ (description "8-pin Dual Flat No-Lead package (DFN), Pitch: 0.50 mm\nNominal width: 3.00 mm\nNominal length: 2.00 mm\nHeight: 0.90mm\nExposed Pad: 1.75 x 1.45 mm\n\nGenerated with librepcb-parts-generator (generate_dfn.py)")
+ (keywords "dfn8,dfn,dual-flat no-leads")
```

Comment:
Are keywords comma separated? So "dual-flat no-leads" is a single keyword?

Reply:
Yes exactly. "DFN" is the abbrevation of "dual flat no-leads", so the whole phrase "dual flat no-leads" is a single keyword. What do you think about this?

Also, you raise a good point: it should be "dual flat no-leads" and not "dual-flat no-leads". That is a bug/typo in all DFN packages. I will correct this in a separate PR.

Reply:
OK. Makes sense to do in a separate PR.